src/viewport/old-viewport.js

```diff
@@ -337,8 +337,8 @@ export default class Viewport {
       this._createMat4(),
       2 * Math.atan((this.height / 2) / this.altitude), // fov in radians
       this.width / this.height,                         // aspect ratio
-      0.1,                                              // near plane
-      this.farZ * 10.0                                  // far plane
+      0.2,                                              // near plane
```

Comment:
> otherwise the instances will not get rendered. changing 0.1 to -0.1 was able to fix those.

I believe Shaojing fixed that by increasing the far plane (the `* 10` multiplier in the code). AFAIK, the near plane must be a positive number. I still kept a `* 2` multiplier which seemed to do the job.